### DISCUSSION TITLE
Ability to define keywords in the metadata

### DISCUSSION THREAD

**Discussion by remi-preghenella** [squidfunk/mkdocs-material#4417] [Ideas]
Hi @squidfunk,

It would be great to be able to define keywords in the metadata of each file and/or the `.meta.yml` files to improve search engine results.

I've changed our `base.html` file to already do it. However, it would be better to have "official" support.

Code change below:
``` 
     {% if page.meta and page.meta.keywords %}
        <meta name="keywords" content="{{ page.meta.keywords }}">
      {% endif %}
```

Thanks for considering the idea!
Rémi

**Comment by squidfunk** [comment]
Thanks for suggesting. The `keywords` meta tag was deprecated in 2018 – it has no use: https://github.com/squidfunk/mkdocs-material/pull/880. I understand that some users still want to have it included, but we're following best practices here, which say: don't use it, it's ignored anyway.

**Reply by remi-preghenella** [reply]
How about `news_keywords`?

Slightly off-topic but also the `robots` one for individual pages?

**Reply by squidfunk** [reply]
You can open a feature request for supporting custom `meta` tags from Markdown. I think we could allow to define `meta` tags from `.meta.yml` (per folder), as well as front matter (per document). This would also allow to add `keywords` and `robot`.
